streaming/base/index.py

```diff
@@ -205,6 +205,14 @@ def _get_min_max_size(start: int, total: int, part: int, num_parts: int):
             worker_max_id = \
                 device_min_id + (worker_max_batch_id + 1) * self.batch_size - max_id_offset - 1
 
+            # Adjustment for last partition.
+            if self.total_samples == worker_max_id:
+                if 0 < worker_min_id:
```

Comment:
Just my preference: sometimes hard to follow `0 < worker_min_id`. Maybe `worker_min_id > 0` is easier to read ?

Reply:
Done